kansha/board/comp.py

```diff
@@ -161,6 +163,10 @@ def many_user_render(h, number):
 
         self.must_reload_search = False
 
+    def reload_save_template_comp(self):
+        self.save_template_comp = component.Component(SaveTemplateTask(self))
+        self.save_template_comp.on_answer(lambda v: self.reload_save_template_comp())
+
     def copy(self, owner, additional_data):
```

Comment:
> Le 2 déc. 2015 à 17:09, Vincent Rialland <notifications@github.com> a écrit :
> 
> In kansha/board/comp.py:
> 
> > @@ -161,6 +163,10 @@ def many_user_render(h, number):
> >  
> >          self.must_reload_search = False
> >  
> > +    def reload_save_template_comp(self):
> > +        self.save_template_comp = component.Component(SaveTemplateTask(self))
> > +        self.save_template_comp.on_answer(lambda v: self.reload_save_template_comp())
> > +
> >      def copy(self, owner, additional_data):
> 
> Some informations need to be propagated through the components tree, for example labels are created at the board level and must be available to cards as they use it.

This copy of informations will disappear in the near future

> These are stored in this dict to be used later.

If you really need to propagate informations down to the cards, explicitly named
them (i.e: `labels` and not `additional_data`)

> —
> Reply to this email directly or view it on GitHub.
>